packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -266,6 +277,15 @@ export const Webchat = forwardRef((props, ref) => {
     props.persistentMenu
   )
 
+  const CustomPersistentMenu = getThemeProperty(
+    'userInput.menu.custom',
+    undefined
+  )
+  const darkBackgroundMenu = getThemeProperty(
```

Comment:
should this start with capitals?

Reply:
It's a variable that can be true or false, should I use capitals in this case?

Reply:
the convention says that boolean names should start or contain "is" "has" (eg. darkBackgroundMenu). However, if in other parts of the code we don't use this convention for the names of the properties or the names of the variables, we should leave it as it is. @marcrabat what do you think?